boxoffice/models/line_item.py

```diff
@@ -152,6 +156,18 @@ def confirm(self):
     def current_assignee(self):
         return self.assignees.filter(Assignee.current == True).one_or_none()  # NOQA
 
+    @property
+    def is_assignable(self):
+        return utcnow().date() <= self.item.event_date
```

Comment:
I just realised this will blow on day 2 of the event, for someone who only shows up on the second day. Are events on a per-day basis?

Also, `utcnow().date()` should be cast into event timezone first. Is that recorded?

Reply:
There is no timezone stored anywhere in boxoffice. There are only the timezone in app config and the user's timezone.